src/components/menu/menu.tsx

```diff
@@ -150,6 +308,301 @@ export class Menu {
         slotElement.assignedElements().forEach(this.setTriggerAttributes);
     }
 
+    private renderLoader = () => {
+        if (!this.loadingSubItems && !this.loading) {
+            return;
+        }
+
+        const cssProperties = this.getCssProperties();
+
+        return (
+            <div
+                style={{
+                    width: cssProperties['--menu-surface-width'],
+                    display: 'flex',
+                    'align-items': 'center',
+                    'justify-content': 'center',
+                    padding: '0.5rem 0',
+                }}
+            >
+                <limel-spinner size="mini" limeBranded={false} />
+            </div>
+        );
+    };
+
+    private renderBreadcrumb = () => {
+        if (!this.selectedMenuItem || this.searchValue) {
+            return;
+        }
+
+        const breadCrumbItems: MenuCrumbItem[] = [];
+        let currentItem = this.selectedMenuItem;
+        while (currentItem) {
+            breadCrumbItems.unshift({
+                text: currentItem.text,
+                icon: {
+                    name: currentItem.icon,
+                    color: currentItem.iconColor,
+                },
+                menuItem: currentItem,
+            });
+            currentItem = currentItem.parentItem;
+        }
+
+        if (!breadCrumbItems.length) {
+            return;
+        }
+
+        return (
+            <limel-breadcrumbs
+                style={{
+                    'border-bottom': 'solid 1px rgb(var(--contrast-500))',
```

Comment:
```suggestion
                    'border-bottom': 'solid 1px rgb(var(--contrast-500))',
                    'flex-shrink': '0',
```

Reply:
fixes this problems
<img width="724" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/98379a5b-a6fd-4eae-a78a-699a7d1cc7c9">

Reply:
Nice find, fixed now.